From d2e8bd06c9178663525b734f9df7dbdb91456243 Mon Sep 17 00:00:00 2001
From: Jan Kratochvil <jan.kratochvil@redhat.com>
Date: Thu, 26 Jul 2012 18:21:49 +0000
Subject: [PATCH] gdb/
 	* auto-load.c (auto_load_info_scripts): Remove immediate_quit increment
 	and decrement.

---
 gdb/ChangeLog   |    5 +++++
 gdb/auto-load.c |    2 --
 2 files changed, 5 insertions(+), 2 deletions(-)

Index: gdb-7.2/gdb/auto-load.c
===================================================================
--- gdb-7.2.orig/gdb/auto-load.c
+++ gdb-7.2/gdb/auto-load.c
@@ -914,11 +914,9 @@ auto_load_info_scripts (char *pattern, i
     {
       struct collect_matching_scripts_data data = { &scripts, language };
 
-      immediate_quit++;
       /* Pass a pointer to scripts as VEC_safe_push can realloc space.  */
       htab_traverse_noresize (pspace_info->loaded_scripts,
 			      collect_matching_scripts, &data);
-      immediate_quit--;
     }
 
   nr_scripts = VEC_length (loaded_script_ptr, scripts);
